pubspec.yaml

```diff
@@ -9,18 +9,17 @@ authors:
 homepage: https://github.com/Workiva/dart_transformer_utils
 dependencies:
   analyzer: ">=0.27.0 <0.33.0"
-  barback: ">=0.15.2 <0.16.0"
-  dart2_constant: ^1.0.0
+  build: ^1.0.0
+  dart2_constant: ^1.0.1
   path: "^1.3.0"
   source_span: "^1.2.0"
 
 dev_dependencies:
   coverage: ">=0.10.0 <0.13.0"
-  dart_dev: ^1.9.2
+  dart_dev: ">=1.9.2 <3.0.0"
   dart_style: ^1.0.9
   dependency_validator: ^1.1.0
-  mockito: ^2.0.0
-  test: ^0.12.32+1
+  test: ">=0.12.32+1 <2.0.0"
 
 environment:
-    sdk: ">=1.24.2 <2.0.0"
+    sdk: ">=2.0.0 <3.0.0"
```

Comment:
#nit This could be ^2.0.0

Reply:
I think pub prevents you from using `^` in the sdk constraint because that syntax isn't recognized by older versions of pub.

Reply:
oh, right